DataManagementSystem/Client/DataManager.py

```diff
@@ -1085,15 +1093,15 @@ def removeFile( self, lfn, force = None ):
       lfns = res['Value']['Successful']
 
       if lfns:
-        log.debug( "Attempting to remove %s files from Storage and Catalogue. Get replicas first" % len( lfns ) )
-        res = self.fc.getReplicas( lfns, True )
+        log.debug( "Attempting to remove %d files from Storage and Catalogue. Get replicas first" % len( lfns ) )
+        res = self.fc.getReplicas( lfns, allStatus = True )
         if not res['OK']:
           errStr = "DataManager.removeFile: Completely failed to get replicas for lfns."
           log.debug( errStr, res['Message'] )
           return res
         lfnDict = res['Value']['Successful']
 
-        for lfn, reason in res['Value'].get( 'Failed', {} ).iteritems():
+        for lfn, reason in res['Value']['Failed'].iteritems():
```

Comment:
At a first glance this seems a bit less safe that the previous version. Why do you want to change?